servers/src/main/java/tachyon/master/next/filesystem/meta/InodeFile.java

```diff
@@ -79,7 +79,6 @@ public FileInfo generateClientFileInfo(String path) {
     ret.isPinned = isPinned();
     ret.isCache = mCache;
     ret.blockIds = getBlockIds();
-    ret.inMemoryPercentage = getInMemoryPercentage();
```

Comment:
Please leave a comment that inMemoryPercentage is filled in by the FileSystemMaster.